2016-06-18 14:55:54 Error: [MissingViewException] View file "/vagrant/cakephp/app/View/Pages/index.ctp" is missing.
Exception Attributes: array (
  'file' => '/vagrant/cakephp/app/View/Pages/index.ctp',
)
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /vagrant/cakephp/lib/Cake/Controller/Controller.php(954): View->render(NULL, NULL)
#2 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(198): Controller->render()
#3 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(PagesController), Object(CakeRequest))
#4 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2016-06-18 17:36:35 Error: Fatal Error (1): Call to a member function user() on a non-object in [/vagrant/cakephp/app/Controller/AppController.php, line 10]
2016-06-18 17:36:35 Error: [FatalErrorException] Call to a member function user() on a non-object
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/vagrant/cakeph...', 10)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/vagrant/cakeph...', 10, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/vagrant/cakeph...', 10, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 17:37:13 Error: Fatal Error (1): Call to a member function user() on a non-object in [/vagrant/cakephp/app/Controller/AppController.php, line 13]
2016-06-18 17:37:13 Error: [FatalErrorException] Call to a member function user() on a non-object
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/vagrant/cakeph...', 13)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/vagrant/cakeph...', 13, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/vagrant/cakeph...', 13, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 17:40:19 Error: [MissingViewException] View file "/vagrant/cakephp/app/View/Users/login.ctp" is missing.
Exception Attributes: array (
  'file' => '/vagrant/cakephp/app/View/Users/login.ctp',
)
Request URL: /users/login
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /vagrant/cakephp/lib/Cake/Controller/Controller.php(954): View->render(NULL, NULL)
#2 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(198): Controller->render()
#3 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2016-06-18 17:44:25 Error: [MissingViewException] View file "/vagrant/cakephp/app/View/Users/index.ctp" is missing.
Exception Attributes: array (
  'file' => '/vagrant/cakephp/app/View/Users/index.ctp',
)
Request URL: /users/index
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /vagrant/cakephp/lib/Cake/Controller/Controller.php(954): View->render(NULL, NULL)
#2 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(198): Controller->render()
#3 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2016-06-18 17:46:20 Error: [PDOException] SQLSTATE[HY000]: General error: 1364 Field 'id' doesn't have a default value
Request URL: /users/signup
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Model/Datasource/DboSource.php(458): PDOStatement->execute(Array)
#1 /vagrant/cakephp/lib/Cake/Model/Datasource/DboSource.php(424): DboSource->_execute('INSERT INTO `ha...', Array)
#2 /vagrant/cakephp/lib/Cake/Model/Datasource/DboSource.php(1012): DboSource->execute('INSERT INTO `ha...')
#3 /vagrant/cakephp/lib/Cake/Model/Model.php(1854): DboSource->create(Object(User), Array, Array)
#4 /vagrant/cakephp/app/Controller/UsersController.php(12): Model->save(Array)
#5 [internal function]: UsersController->signup()
#6 /vagrant/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#7 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#9 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2016-06-18 17:46:24 Error: [PDOException] SQLSTATE[HY000]: General error: 1364 Field 'id' doesn't have a default value
Request URL: /users/signup
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Model/Datasource/DboSource.php(458): PDOStatement->execute(Array)
#1 /vagrant/cakephp/lib/Cake/Model/Datasource/DboSource.php(424): DboSource->_execute('INSERT INTO `ha...', Array)
#2 /vagrant/cakephp/lib/Cake/Model/Datasource/DboSource.php(1012): DboSource->execute('INSERT INTO `ha...')
#3 /vagrant/cakephp/lib/Cake/Model/Model.php(1854): DboSource->create(Object(User), Array, Array)
#4 /vagrant/cakephp/app/Controller/UsersController.php(12): Model->save(Array)
#5 [internal function]: UsersController->signup()
#6 /vagrant/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#7 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#9 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2016-06-18 17:46:40 Error: [PDOException] SQLSTATE[HY000]: General error: 1364 Field 'id' doesn't have a default value
Request URL: /users/signup
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Model/Datasource/DboSource.php(458): PDOStatement->execute(Array)
#1 /vagrant/cakephp/lib/Cake/Model/Datasource/DboSource.php(424): DboSource->_execute('INSERT INTO `ha...', Array)
#2 /vagrant/cakephp/lib/Cake/Model/Datasource/DboSource.php(1012): DboSource->execute('INSERT INTO `ha...')
#3 /vagrant/cakephp/lib/Cake/Model/Model.php(1854): DboSource->create(Object(User), Array, Array)
#4 /vagrant/cakephp/app/Controller/UsersController.php(12): Model->save(Array)
#5 [internal function]: UsersController->signup()
#6 /vagrant/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#7 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#9 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2016-06-18 17:50:48 Error: [MissingTableException] Table users for model User was not found in datasource default.
Exception Attributes: array (
  'table' => 'users',
  'class' => 'User',
  'ds' => 'default',
)
Request URL: /users/signup
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Model/Model.php(3551): Model->setSource('users')
#1 /vagrant/cakephp/lib/Cake/Model/Model.php(3066): Model->getDataSource()
#2 /vagrant/cakephp/lib/Cake/Model/Model.php(2979): Model->_findCount('before', Array)
#3 /vagrant/cakephp/lib/Cake/Model/Model.php(2914): Model->buildQuery('count', Array)
#4 /vagrant/cakephp/lib/Cake/Model/Model.php(3334): Model->find('count', Array)
#5 [internal function]: Model->isUnique(Array, Array)
#6 /vagrant/cakephp/lib/Cake/Model/Validator/CakeValidationRule.php(275): call_user_func_array(Array, Array)
#7 /vagrant/cakephp/lib/Cake/Model/Validator/CakeValidationSet.php(135): CakeValidationRule->process('username', Array, Array)
#8 /vagrant/cakephp/lib/Cake/Model/ModelValidator.php(267): CakeValidationSet->validate(Array, false)
#9 /vagrant/cakephp/lib/Cake/Model/ModelValidator.php(99): ModelValidator->errors(Array)
#10 /vagrant/cakephp/lib/Cake/Model/Model.php(3367): ModelValidator->validates(Array)
#11 /vagrant/cakephp/lib/Cake/Model/Model.php(1758): Model->validates(Array)
#12 /vagrant/cakephp/app/Controller/UsersController.php(12): Model->save(Array)
#13 [internal function]: UsersController->signup()
#14 /vagrant/cakephp/lib/Cake/Controller/Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#15 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#16 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#17 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#18 {main}
2016-06-18 17:54:15 Error: [MissingViewException] View file "/vagrant/cakephp/app/View/Users/mypage.ctp" is missing.
Exception Attributes: array (
  'file' => '/vagrant/cakephp/app/View/Users/mypage.ctp',
)
Request URL: /users/mypage
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /vagrant/cakephp/lib/Cake/Controller/Controller.php(954): View->render(NULL, NULL)
#2 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(198): Controller->render()
#3 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2016-06-18 17:54:26 Error: [MissingViewException] View file "/vagrant/cakephp/app/View/Users/mypage.ctp" is missing.
Exception Attributes: array (
  'file' => '/vagrant/cakephp/app/View/Users/mypage.ctp',
)
Request URL: /users/mypage
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/View/View.php(468): View->_getViewFileName(NULL)
#1 /vagrant/cakephp/lib/Cake/Controller/Controller.php(954): View->render(NULL, NULL)
#2 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(198): Controller->render()
#3 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2016-06-18 18:02:00 Error: Fatal Error (4): syntax error, unexpected $end in [/vagrant/cakephp/app/View/Elements/pc/header.ctp, line 12]
2016-06-18 18:02:00 Error: [FatalErrorException] syntax error, unexpected $end
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/vagrant/cakeph...', 12)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/vagrant/cakeph...', 12, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/vagrant/cakeph...', 12, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 18:03:01 Error: Fatal Error (4): syntax error, unexpected $end in [/vagrant/cakephp/app/View/Elements/pc/header.ctp, line 12]
2016-06-18 18:03:01 Error: [FatalErrorException] syntax error, unexpected $end
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/vagrant/cakeph...', 12)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/vagrant/cakeph...', 12, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/vagrant/cakeph...', 12, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 18:03:11 Error: Fatal Error (4): syntax error, unexpected $end in [/vagrant/cakephp/app/View/Elements/pc/header.ctp, line 12]
2016-06-18 18:03:11 Error: [FatalErrorException] syntax error, unexpected $end
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/vagrant/cakeph...', 12)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/vagrant/cakeph...', 12, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/vagrant/cakeph...', 12, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 18:03:21 Error: Fatal Error (4): syntax error, unexpected $end in [/vagrant/cakephp/app/View/Elements/pc/header.ctp, line 12]
2016-06-18 18:03:21 Error: [FatalErrorException] syntax error, unexpected $end
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/vagrant/cakeph...', 12)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/vagrant/cakeph...', 12, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/vagrant/cakeph...', 12, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 18:03:40 Error: Fatal Error (4): syntax error, unexpected ':', expecting ';' in [/vagrant/cakephp/app/View/Elements/pc/header.ctp, line 10]
2016-06-18 18:03:40 Error: [FatalErrorException] syntax error, unexpected ':', expecting ';'
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/vagrant/cakeph...', 10)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/vagrant/cakeph...', 10, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/vagrant/cakeph...', 10, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 18:03:42 Error: Fatal Error (4): syntax error, unexpected ':', expecting ';' in [/vagrant/cakephp/app/View/Elements/pc/header.ctp, line 10]
2016-06-18 18:03:42 Error: [FatalErrorException] syntax error, unexpected ':', expecting ';'
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/vagrant/cakeph...', 10)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/vagrant/cakeph...', 10, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/vagrant/cakeph...', 10, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 18:03:44 Error: Fatal Error (1): Call to a member function loggedIn() on a non-object in [/vagrant/cakephp/app/View/Elements/pc/header.ctp, line 4]
2016-06-18 18:03:44 Error: [FatalErrorException] Call to a member function loggedIn() on a non-object
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/vagrant/cakeph...', 4)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/vagrant/cakeph...', 4, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/vagrant/cakeph...', 4, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 18:05:10 Error: Fatal Error (1): Call to a member function allow() on a non-object in [/vagrant/cakephp/app/Controller/PagesController.php, line 8]
2016-06-18 18:05:10 Error: [FatalErrorException] Call to a member function allow() on a non-object
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/vagrant/cakeph...', 8)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/vagrant/cakeph...', 8, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/vagrant/cakeph...', 8, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 18:07:52 Error: [MissingActionException] Action PagesController::display() could not be found.
Exception Attributes: array (
  'controller' => 'PagesController',
  'action' => 'display',
)
Request URL: /pages/index
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#1 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(PagesController), Object(CakeRequest))
#2 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2016-06-18 18:08:09 Error: [MissingActionException] Action PagesController::display() could not be found.
Exception Attributes: array (
  'controller' => 'PagesController',
  'action' => 'display',
)
Request URL: /pages/index
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#1 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(PagesController), Object(CakeRequest))
#2 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2016-06-18 18:14:17 Error: Fatal Error (4): syntax error, unexpected '[', expecting ',' or ';' in [/vagrant/cakephp/app/View/Elements/pc/header.ctp, line 5]
2016-06-18 18:14:17 Error: [FatalErrorException] syntax error, unexpected '[', expecting ',' or ';'
Request URL: /
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Error/ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', '/vagrant/cakeph...', 5)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', '/vagrant/cakeph...', 5, Array)
#2 /vagrant/cakephp/lib/Cake/Core/App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', '/vagrant/cakeph...', 5, Array)
#3 /vagrant/cakephp/lib/Cake/Core/App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2016-06-18 18:18:30 Error: [MissingControllerException] Controller class LoginController could not be found.
Exception Attributes: array (
  'class' => 'LoginController',
  'plugin' => NULL,
)
Request URL: /login
Stack Trace:
#0 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2016-06-18 18:21:07 Error: [MissingActionException] Action UsersController::mypage3() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'mypage3',
)
Request URL: /users/mypage3
Stack Trace:
#0 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#1 /vagrant/cakephp/lib/Cake/Routing/Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#2 /vagrant/cakephp/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
